src/guidescanpy/flask/blueprints/web.py

```diff
@@ -66,6 +66,10 @@ def grna_design():
             form_data["ce-bounds-l"] = float(form["txtFilterAboveCE"])
             form_data["ce-bounds-u"] = 1.0
 
+        if form.get("checkFilterAboveGCContent", "off") == "on":
+            form_data["gc-bounds-l"] = float(form["txtFilterAboveGCContent"])
+            form_data["gc-bounds-u"] = 1.0
```

Comment:
Thanks for all the changes! I think this assignment to `gc-bounds-u` can go away without affecting anything.